eZ/Publish/Core/Persistence/Cache/Tests/AbstractCacheHandlerTest.php

```diff
@@ -27,7 +27,7 @@
      * @param array|null $tags
      * @param string|null $key
      */
-    final public function testUnCachedMethods(string $method, array $arguments, array $tags = null, string $key = null)
+    final public function testUnCachedMethods(string $method, array $arguments, array $tags = null, string $key = null, array $additionalCalls = [])
```

Comment:
typically move test out of the standard `providerForUnCachedMethods` in such cases in favor of custom test. Unless you of course see a recurring generic need for this.